mkdocs.yml

```diff
@@ -47,13 +61,23 @@ theme:
 plugins:
 - mkdocs-jupyter:
       include_source: True
-      include: ["pk_wishful_workflows.ipynb"]
+      include: [
+        "pk_wishful_workflows.ipynb",
```

Comment:
I agree the wishful notebook can be repurposed as a mini-tutorial, but it should be completely rewritten. Shall we make a new issue for that?

Reply:
issue #111